packages/mobile/src/icons/CopyIcon.tsx

```diff
@@ -0,0 +1,21 @@
+import * as React from 'react'
+import { Image } from 'react-native'
+import { copyIcon } from 'src/images/Images'
+
+interface Props {
+  height?: number
+  width?: number
+}
+
+export default class CopyIcon extends React.PureComponent<Props> {
+  static defaultProps = {
+    height: 24,
+    width: 24,
+  }
```

Comment:
This kind of wrapper should only be used for SVG icons.
Fixed resolution icons won't look nice if not passed their native resolution.